src/Illuminate/Encryption/Encrypter.php

```diff
@@ -112,10 +112,10 @@ public function decrypt($payload)
      *
      * @throws \Exception
      */
-    protected function mcryptDecrypt($value, $iv)
+    protected function opensslDecrypt($value, $iv)
     {
         try {
-            return mcrypt_decrypt($this->cipher, $this->key, $value, $this->mode, $iv);
+            return openssl_decrypt($value, $this->cipher, $this->key, OPENSSL_RAW_DATA | OPENSSL_ZERO_PADDING, $iv);
```

Comment:
We need to assign this to a variable and check it's not `=== false` too before returning it.